src/controllers/dynakube/version/activegate.go

```diff
@@ -11,18 +11,18 @@ import (
 type activeGateUpdater struct {
 	dynakube *dynatracev1beta1.DynaKube
 	dtClient dtclient.Client
-	hashFunc ImageHashFunc
+	digestFunc ImageDigestFunc
 }
 
 func newActiveGateUpdater(
 	dynakube *dynatracev1beta1.DynaKube,
 	dtClient dtclient.Client,
-	hashFunc ImageHashFunc,
+	hashFunc ImageDigestFunc,
```

Comment:
```suggestion
	digestFunc ImageDigestFunc,
```

Reply:
Same goes for the other updaters as well if you think about commiting this suggestion